docs/design-system.md

```diff
@@ -0,0 +1,73 @@
+
+### Some caveats
+
+1. Buttons!
+
+- Figma has 3 types of buttons: Primary, Secondary, Tertiary. The difference between a secondary button and a tertiary button is the active state.
+- There’s a 4th type of button in the design system called Plain - this button type has no styles, it’s just a reset button without any outline or background color that can be used when we need something to semantically be a button but the designs disagree. This shouldn’t happen often and should be a bit of an escape hatch.
+- Button variants in figma are square and rectangle. The additional variants of circle and icon are added outside of figma because there are occasions in the editor and dashboard where there are no shared button styles but the element is expected to behave as a button. By having a button variant of icon we can eliminate the need of having specific exportable buttons like `ArrowDown`.
+- Button types, sizes, and variants (circle, rectangle, square, icon) are all available as constants and importable via `BUTTON_TYPES`, `BUTTON_SIZES`, `BUTTON_VARIANTS` so that we don’t have to worry about miscellaneous strings.
+- Every button combination has a [demo in storybook](https://google.github.io/web-stories-wp/storybook/?path=/story/designsystem-components-button--default).
+
+1. Typography
+
+- [design-system/typography](https://github.com/google/web-stories-wp/tree/main/assets/src/design-system/components/typography) has the only nested content. This follows the foundation set up in figma of Display, Headline, Label, Paragraph, Link.
+- Each typography type has a set of presets that are stored in the theme and matching constants ([theme/constants/typography](https://github.com/google/web-stories-wp/blob/main/assets/src/design-system/theme/constants/typography.js)).
+- The typography components are `<Display />`, `<Headline />`, `<Link />`, `<Text />`
+- Each is a styled component and can be passed whatever element you want it to be with `as` or `forwardedAs` depending on your situation.
+- These are meant to decrease our need to individually style text elements and ensure consistency.
+- There’s a demo for every [type setting in storybook](https://google.github.io/web-stories-wp/storybook/?path=/story/designsystem-components-typography-display--default).
+
+1. Circular dependencies
```

Comment:
```suggestion
3. Circular dependencies
```

Reply:
thanks, when i ran this threw notion to grab markdown by default i didn't notice the numbers reset.